CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -26,6 +26,29 @@ local PlayerHospital = _G["PlayerHospital"]
 local num_sitting_ratios = 15 -- Number of stored recent sitting ratio measurements.
 local ratio_interval = 2 -- Measurement interval in days.
 
+strict_declare_global "staff_initials_cache"
+staff_initials_cache = {}
+
+local function our_concat(t)
```

Comment:
For some reason I constantly run into this function. (Maybe you are moving it all the time :p )

It's not a big problem, but I think eventually it should be a global function with a better name and some documentation.
The question is 'where', but I have no answer to that.

@TheCycoONE  Maybe you have some idea?

Reply:
Utility.lua?

Reply:
That would be indeed the obvious spot.

I looked through it yesterday, but it doesn't have any string other function, so if you don't know that the function exists, you will never find it. Obviously, that's the case now as well, but I was hoping to avoid that.

There is also `strings.lua`, but that is also not a good spot it seemed on my first scan.

Ah well, we can always keep it around like it is :)

Reply:
This isn't strictly resolved, but I don't mind solving it later.